rasa/core/policies/memoization.py

```diff
@@ -159,7 +160,7 @@ def _create_feature_key(self, states: List[State]) -> Text:
         # represented as dictionaries have the same json strings
         # quotes are removed for aesthetic reasons
         feature_str = json.dumps(states, sort_keys=True).replace('"', "")
-        if self.ENABLE_FEATURE_STRING_COMPRESSION:
+        if False:  # self.ENABLE_FEATURE_STRING_COMPRESSION:
```

Comment:
Why remove this in this PR?

Reply:
🤦🏻 ah thanks for catching that. Was looking at how the states got cached when writing tests and forgot to unset. Reverted now.